src/Controller/EventNodeController.php

```diff
@@ -0,0 +1,83 @@
+<?php
+/**
+ * @file
+ */
+
+namespace Drupal\popapi\Controller;
+
+use Drupal\node\Entity\Node;
+
+/**
+ * Class EventNodeController
+ * @package Drupal\popapi\Controller
+ */
+class EventNodeController {
+
+  /**
+   * @return array|int
+   */
+  public static function loadLatestEventDate() {
+    $query = \Drupal::entityQuery('node');
+    $query->condition('status', 1);
+    $query->condition('type', 'events');
+    $query->sort('field_eventbrite_event_created', 'ASC');
```

Comment:
Can have "field_eventbrite_event_created" as a const of the class. Can also have "events" (line 22) as a const of bundle for the class